syntax = "proto3";

package unary;

service Unary{
    //for authentification
    rpc AuthentificateUser(User) returns(UserResponse) {}
    //for calculations
    rpc GetServerResponse(Message) returns (MessageResponse) {}
}

message Message{
    string message = 1;
}

message MessageResponse{
    string message = 1;
    bool received = 2;
}

message User{
    string login = 1;
    string password = 2;
}

message UserResponse{
    string message = 1;
    bool logged_in = 2;
}
